Source/DafnyCore/Auditor/Assumption.cs

```diff
@@ -25,22 +25,35 @@ bool allowedInLibraries
     allowedInLibraries: false);
   public static AssumptionDescription ExternFunction = new(
     issue: "Function with [{:extern}] attribute.",
-    mitigation: "Turn into a [method] with [modifies *] or test thoroughly for lack of side effects.",
-    mitigationIETF: "SHOULD use [method] with [modifies *] instead.",
-    isExplicit: false,
-    allowedInLibraries: false);
-  public static AssumptionDescription ExternWithPrecondition = new(
-    issue: "Declaration with [{:extern}] has a requires clause.",
-    mitigation: "Test any external callers (maybe with [/testContracts]).",
-    mitigationIETF: "MUST test any external callers.",
-    isExplicit: false,
-    allowedInLibraries: false);
-  public static AssumptionDescription ExternWithPostcondition = new(
-    issue: "Declaration with [{:extern}] has a ensures clause.",
-    mitigation: "Test external callee (maybe with [/testContracts]).",
-    mitigationIETF: "MUST test external callee.",
+    mitigation: "Turn into a [method] with [modifies {}] or test thoroughly for lack of side effects.",
```

Comment:
```suggestion
    mitigation: "Turn into a [method] with [modifies {}] and test thoroughly for lack of side effects.",
```

Reply:
Turning it into a method allows for non-determinism that may occur, but if we have `modifies {}` then we'll really want to know that it has no side effects.